### ISSUE TITLE
Some problem about office-31 dataset

### ISSUE THREAD

**Issue by yujiapingff** [Hongbin98/ProCA#3]
I use the OH_adapt_2_target.py to train the pretrain model on the office-31 dataset, but I cannot reach the result accuracy which showing in the paper, is there any modification should apply on this file to make it suit for office-31 dataset.

**Comment by Hongbin98** [comment]
Thanks for your attention of our research.
Please check if you are using the same settings as our paper mentioned (Implementation details and Data splits).
